app.json

```diff
@@ -85,7 +85,8 @@
         "backgroundColor": "#ffffff"
       },
       "package": "dev.nedaa.android",
-      "blockedPermissions": ["ACTIVITY_RECOGNITION"]
+      "blockedPermissions": ["ACTIVITY_RECOGNITION"],
+      "permissions": ["READ_MEDIA_AUDIO", "READ_EXTERNAL_STORAGE", "WRITE_EXTERNAL_STORAGE"]
```

Comment:
With my testing, it seems we don't need `WRITE_EXTERNAL_STORAGE`, everything works fine, let me know if we need it for something else, otherwise we can remove it.